### DISCUSSION TITLE
Box boundry collision import option

### DISCUSSION THREAD

**Discussion by RexTheWho** [godotengine/godot-proposals#6599] [Editor]
Reading through the 'importing 3d scenes' documentation its very annoying that there's ways to get complicated convex collisions but seemingly nothing for just a simple box bounds option. The docs also mention empties being used but I never managed to get those to show anything but node3ds, not whats important just as a tangent.

There should really be something like `-colbox` tag for imports that grabs the boundary of a mesh and gives it a box that surrounds it.
Additionally `-colboxonly` (for multiple simple boxes) would be good also for complicated meshes like doorframes that has a cavity but you don't really want to use a convex collision.

Bit of a bummer that the workflow for setting up collisions in 3D is a bit of an imprecise tedium in the editor compared to working in the modelling tool the actual models export from.

(If there is something Ive genuinely missed please let me know! ...Because I haven't found it.)

**Comment by AThousandShips** [comment]
Related #5868, #3155
